package.json

```diff
@@ -516,12 +516,14 @@
 		"@typescript-eslint/eslint-plugin": "^5.11.0",
 		"@typescript-eslint/parser": "^5.11.0",
 		"esbuild": "^0.11.12",
-		"eslint": "^7.15.0",
+		"eslint": "^7.32.0",
+		"eslint-config-prettier": "^8.5.0",
 		"fast-xml-parser": "^3.20.0",
 		"glob": "^7.1.7",
 		"js-yaml": "^4.1.0",
 		"mocha": "^8.1.3",
 		"npm-license-crawler": "^0.2.1",
+		"prettier": "2.6.2",
```

Comment:
Not a big thing but if you want to add the ability to shift up rather have a strict peg, that would be nice.

Reply:
I think it makes sense to pin `prettier` in the package.json to the same version in the pre-commit config. As long as we remember to bump this whenever it gets bumped in the pre-commit config.

Reply:
Hmm it's a bit of a bummer to have to specify prettier version in two places.  I wonder if that's avoidable? 🤔

Reply:
Ok I've now pinned everything in package.json that's specified in the pre-commit config, but that seems not great.  I wonder if there's a better way

Reply:
I'll open an issue on their repo and see what comes of it. I think that duplicating it is the path forward for now.

Reply:
It looks like this is the resolution. The tldr; is to rely on a [repository local hook](https://pre-commit.com/#repository-local-hooks)and throw the check into an npm or yarn script.

https://github.com/pre-commit/pre-commit/issues/945#issuecomment-527234424

Reply:
Ok, given that currently `eslint` is already run in CI, the `eslint --fix` isn't buying us a ton, so probably not worth the effort.  Let's remove the `eslint` fixer for now, and wait until we start using it to remove unused imports in #640.  I think it's ok to leave `prettier` pinned until then as well.  It was mainly the pinned eslint typescript parser plugin making me nervous

Sound good?